examples/3rd-party-oauth-login/README.md

```diff
@@ -0,0 +1,103 @@
+# 3rd Party OAuth Login
+
+This app allows you to login to a 3rd party service using (you will need to provide OAuth URL) and tracks the logged in status via local storage.
```

Comment:
```suggestion
This app allows you to login to a 3rd party service using (you will need to provide a OAuth URL) and track the logged in status via local storage.
```

Reply:
changed to the following:

This app allows you to login to a 3rd party service using OAuth (you will need to provide an OAuth URL) and tracks the logged in status via local storage.

Reply:
Still wrong I think. 
This app allows you to ...
AND tracks

Maybe it should be 'and IT tracks'?

Reply:
the IT is implied :) 

It is grammatically correct as is now, but I can change it to "IT tracks".

Reply:
@mettin I split it into two sentences for easier comprehension. 


This app allows you to login to a 3rd party service using OAuth (you will need to provide an OAuth URL). The app tracks if the user
has logged in or not via [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage).